<launch>
    <arg name="debug" default="false"/>

    <node pkg="bigbang_eurobot" type="global_planer" name="global_planer" output="screen" respawn="true" respawn_delay="1"> 
        <param name = "debug" value = "$(arg debug)"/>
    </node>

</launch>